docs/getting-started/slack.md

```diff
@@ -68,5 +71,12 @@ capabilities and how to interact with it.
   <iframe width="560" height="315" src="https://www.youtube.com/embed/B_x43nPoDH4" frameborder="0" allowfullscreen></iframe>
 </div>
 
+For a complete list of the things the Atomist Bot can do, send the bot
+a `help` message.
+
+<div class="ss-container">
+  <img src="../images/bot-help.png" alt="Success" class="ss-large">
+</div>
+
 Click Next in the bottom-right of this page to continue the Getting
```

Comment:
I want to make sure they know that the "Next" link exists and where it is. Allowing them to click right here kind of defeats the purpose.